pkg/envoy/proxy.go

```diff
@@ -134,12 +134,12 @@ func (e *envoy) args(fname string, bootstrapConfig string) []string {
 	if e.ProxyConfig.LogAsJSON {
 		startupArgs = append(startupArgs,
 			"--log-format",
-			`{"level":"%l","time":"%Y-%m-%dT%T.%fZ","scope":"envoy %n","msg":"%j"}`,
+			`{"level":"%l","time":"%Y-%m-%dT%T.%fZ","scope":"envoy %n","msg":"%j","loc":"%g:%#","tid":"%t"}`,
```

Comment:
`caller` is the key used in Istio json log format (eg `{"level":"info","time":"2022-10-18T17:59:28.556271Z","caller":"cobra@v1.5.0/command.go:872","msg":"some log","thread":1234}`)

Reply:
Done.